spec/models/invoice_item_spec.rb

```diff
@@ -9,4 +9,10 @@
   describe "enum" do
     it { should define_enum_for(:status).with_values(["packaged", "pending", "shipped"]) }
   end
+
+  describe "validations" do
+    it { should validate_presence_of :quantity }
+    it { should validate_presence_of :unit_price }
+    it { should validate_presence_of :status }
```

Comment:
I think we want to test the validation for status a little further here to make sure it comes in as one of the three pre-approved values, right?